safekeeper/src/http/routes.rs

```diff
@@ -155,6 +238,14 @@ pub fn make_router(conf: SafeKeeperConf) -> RouterBuilder<hyper::Body, ApiError>
             timeline_status_handler,
         )
         .post("/v1/timeline", timeline_create_handler)
+        .delete(
+            "/v1/tenant/:tenant_id/timeline/:timeline_id/delete_force",
```

Comment:
I'd drop `delete_force` part. It is strange to use delete method and repeat that in URL. And on force, currently we don't have a choice at all; if we have it later, we can specify in body the behaviour. Not very RESTy, but anyway.

I was also concerned about URL inconsistency with other methods, but that makes API closer to pageserver,  so ok, let's keep it this way and at some point make it fully identical.